account-management-app/src/main/java/org/duracloud/account/app/controller/DuracloudMillForm.java

```diff
@@ -84,4 +101,107 @@ public void setAuditLogSpaceId(String auditLogSpaceId) {
         this.auditLogSpaceId = auditLogSpaceId;
     }
 
+    public String getAuditQueueType() {
+        return auditQueueType;
+    }
+
+    public void setAuditQueueType(String auditQueueType) {
+        this.auditQueueType = auditQueueType;
+    }
+
+    public String getRabbitmqHost() {
+        return rabbitmqHost;
+    }
+
+    public void setRabbitmqHost(String rabbitmqHost) {
+        if (settingRabbitMQ()) {
+            if (rabbitmqHost == null) {
+                throw nullConstraintViolationException();
+            } else {
+                this.rabbitmqHost = rabbitmqHost;
+            }
+        } else {
+            this.rabbitmqHost = null;
+        }
+    }
+
+    public Integer getRabbitmqPort() {
+        return rabbitmqPort;
+    }
+
+    public void setRabbitmqPort(Integer rabbitmqPort) {
+        if (settingRabbitMQ()) {
+            if (rabbitmqPort == null) {
+                throw nullConstraintViolationException();
+            } else {
+                this.rabbitmqPort = rabbitmqPort;
+            }
+        } else {
+            this.rabbitmqPort = 5672;
+        }
+    }
+
+    public String getRabbitmqVhost() {
+        return rabbitmqVhost;
+    }
+
+    public void setRabbitmqVhost(String rabbitmqVhost) {
+        if (settingRabbitMQ()) {
+            if (rabbitmqVhost == null) {
+                throw nullConstraintViolationException();
+            } else {
+                this.rabbitmqVhost = rabbitmqVhost;
+            }
+        } else {
+            this.rabbitmqVhost = "/";
```

Comment:
Is this a standard/default Rabbit MQ vhost? Is this the value most people would use?

Reply:
Yes, any new installations of RabbitMQ will have a single vhost, `/`. However, most users would probably customise this, especially if they are using their RabbitMQ cluster for other purposes, which is likely.